playbook/vite.config.ts

```diff
@@ -20,6 +20,7 @@ export default defineConfig({
       compress: false,
     },
     rollupOptions: {
+      preserveEntrySignatures: 'strict',
```

Comment:
So for this: playbook-doc.js was doing more than feeding the website, it imported every kit for the examples and because Rollup builds the npm package and sees a real module (playbook-doc.js) using each kit, it kept all those exports alive even though playbook.js only re‑exports them. Once we stopped building playbook-doc.js, the bundler no longer saw any “in-project” usage of most kits, so it tree‑shook them out of playbook.js. Adding preserveEntrySignatures: 'strict' tells Rollup to keep the whole export surface even if nothing inside the build references it, restoring pre-change behavior.